generated/nifake/session.py

```diff
@@ -115,7 +115,7 @@ class _SessionBase(object):
     An property of type float with read/write access.
```

Comment:
All these say "An property", should say "A property". I bet this happened due to attribute->property replacement. If that's right, please open a separate issue about it.

Reply:
It is. #808